tdrs-frontend/src/components/Welcome/Welcome.jsx

```diff
@@ -40,25 +30,64 @@ function Welcome() {
   }
 
   return (
-    <GridContainer className="welcome">
-      <Grid row>
-        <Grid col={6} className="left">
-          <h1>
-            Welcome to TDRS!
-            <span role="img" aria-label="wave" aria-hidden="true">
-              {' '}
-              👋
-            </span>
-          </h1>
-        </Grid>
-        <Grid col={6} className="right">
-          <Button type="button" size="big" onClick={handleClick}>
-            Sign in with Login.gov
-          </Button>
+    <main id="main-content">
+      <section className="usa-hero" aria-label="Introduction">
+        <GridContainer>
+          <Grid>
+            <div className="usa-hero__callout">
+              <h1 className="usa-hero__heading">
+                <span className="usa-hero__heading--alt">
+                  Sign into TANF Data Portal
+                </span>
+              </h1>
+              <p>
+                Our vision is to build a new, secure, web based data reporting
+                system to improve the federal reporting experience for TANF
+                grantees and federal staff. The new system will allow grantees
+                to easily submit accurate data and be confident that they have
+                fulfilled their reporting requirements.
+              </p>
+              <Button type="button" size="big" onClick={handleClick}>
+                Sign in with Login.gov
+              </Button>
+            </div>
+          </Grid>
+        </GridContainer>
+      </section>
+
+      <section className="grid-container usa-section">
+        <Grid row gap>
+          <Grid tablet={{ col: 4 }}>
+            <h2 className="font-heading-xl margin-top-0 tablet:margin-bottom-0">
+              Featured TANF Resources
+            </h2>
+            <p>Questions about TANF data?</p>
+            <p>Email: tanfdata@acf.hhs.gov</p>
+          </Grid>
+          <Grid tablet={{ col: 8 }}>
+            <img
+              className="usa-media-block__img"
+              src={circleImg}
+              style={{ height: '120px' }}
+              alt="Alt text"
+            />
+            <img
+              className="usa-media-block__img"
+              src={circleImg}
+              style={{ height: '120px' }}
+              alt="Alt text"
+            />
+            <img
+              className="usa-media-block__img"
+              src={circleImg}
+              style={{ height: '120px' }}
+              alt="Alt text"
```

Comment:
same comment as above for all "Alt text" references.

Reply:
these were removed